### PR TITLE
[bsp][lpc55sxx] update keil project files

### PR DESCRIPTION
**Pull request by yandld**
## 拉取/合并请求描述：(PR description)

[

#### 为什么提交这份PR (why to submit this PR)
LPC5Sxx NXP support, update Keil project according to rtt source change

#### 在什么测试环境下测试通过 (what is the test environment)

]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 代码是高质量的 Code in this PR is of high quality
- [x] 已经使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

File: bsp/lpc55sxx/lpc55s69_nxp_evk/rtconfig.h [hunk 1]
```diff
@@ -137,6 +137,18 @@
...
+#define ULOG_OUTPUT_LVL 7
+#define ULOG_ASSERT_ENABLE
+#define ULOG_LINE_BUF_SIZE 128
+
+/* log format */
+
+#define ULOG_OUTPUT_FLOAT
+#define ULOG_OUTPUT_LEVEL
+#define ULOG_OUTPUT_TAG
+#define ULOG_BACKEND_USING_CONSOLE
```
Comment (mysterywolf) [inline bsp/lpc55sxx/lpc55s69_nxp_evk/rtconfig.h:151]: 请不要默认开启这些功能

File: bsp/lpc55sxx/lpc55s69_nxp_evk/rtconfig.h [hunk 2]
```diff
@@ -183,6 +195,8 @@
 
 /* enhanced kernel services */
 
+#define PKG_USING_RT_VSNPRINTF_FULL
+#define PKG_USING_RT_VSNPRINTF_FULL_LATEST_VERSION
```
Comment (mysterywolf) [inline bsp/lpc55sxx/lpc55s69_nxp_evk/rtconfig.h:199]: 不要开启软件包

**Review by mysterywolf — CHANGES_REQUESTED** [review]

**Comment by yandld** [comment]
已修改

File: bsp/lpc55sxx/lpc55s69_nxp_evk/.config [hunk 3]
```diff
@@ -499,7 +491,15 @@ CONFIG_RT_LIBC_DEFAULT_TIMEZONE=8
 #
 # CONFIG_PKG_USING_RT_MEMCPY_CM is not set
 # CONFIG_PKG_USING_RT_KPRINTF_THREADSAFE is not set
-# CONFIG_PKG_USING_RT_VSNPRINTF_FULL is not set
+CONFIG_PKG_USING_RT_VSNPRINTF_FULL=y
```
Comment (supperthomas) [inline bsp/lpc55sxx/lpc55s69_nxp_evk/.config:494]: CONFIG_PKG_USING_RT_VSNPRINTF_FULL  这里这个软件包确定需要加到工程里面吗？

**Review by supperthomas — APPROVED** [review]

**Review by mysterywolf — APPROVED** [review]

**Pull request merged**